### DISCUSSION TITLE
Starting with web first. Good idea?

### DISCUSSION THREAD

**Discussion by kennym** [nandorojo/solito#89] [Q&A] [answered]
After reading the documentation it feels like it might be best to create the React Native app first, and then think about web.

Is that true? If not, how would you go about building for web/mobile web first, and then native? What would be some things to keep in mind?

**Comment by nandorojo** [comment, marked as answer]
Not really. Have you seen the starter?

I suppose you can start focusing more on one, but it should be easy to do both in tandem.

**Reply by kennym** [reply]
Is it possible to skip `react-native-web` when using solito for the Next app? I feel like by going the `react-native-web` route we lose access to a lot of popular packages for web.

Would I by not going for `react-native-web` sacrifice code re-usability?

What do you recommend to one to adopt the `react-native-web` mentality?

**Reply by nandorojo** [reply]
I mean, using RNW just means using RN. So if you don’t use RN it kinda defeats the point on Web of Solito, since you can’t share code. What’s an example of something you’d miss out on?

**Reply by nandorojo** [reply]
one place to just use Web might be marketing pages, but for the rest, I would use RN and find the right libraries to make it work.

**Reply by nandorojo** [reply]
technically the next app can use just web code, but it won’t work on native
